### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aditsrin**
Update getting started guidelines with the latest changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,41 +20,16 @@ With Actions Packages, you can reference actions by their version and feel confi
...
-	$ gh action-release create
-	$ gh action-release create -r github/js-action -c minor -t "release title" -p n
-	$ gh action-release create -r github/js-action -c minor -t "release title" -n "release notes" -p n
-	$ gh action-release create --repo github/js-action --changetype minor --title "release title" --releasenotes "release notes" --prerelease n
-```
-
-- The create release command above returns the link of the release workflow run, visit the link to track progress of package publishing. A package with with repo name and a generated semver compliant tag would be published to GHCR, 
-
-<img width="1400" alt="image" src="https://user-images.githubusercontent.com/13884596/190091843-d739d30e-1406-451a-a00d-98c09a4b74d5.png">
+- After adding of release.yml file into, a user can create packages for their action repo and publish them to GHCR by creating releases with some conditions on the `gitTag` used.
```
Comment (vinaykanamarlapudi) [inline README.md:23]: Can be reframed as : After adding release.yml, action packages can be created and published to GHCR by following the normal release procedure either using Repo Releases GUI or CLI.

File: README.md [hunk 2]
```diff
@@ -20,41 +20,16 @@ With Actions Packages, you can reference actions by their version and feel confi
...
+- The `gitTag` used for release creation should follow the guidlines of being semver compliant as mentioned in the release creation page. User should follow these guidlines :
```
Comment (vinaykanamarlapudi) [inline README.md:24]: Can be reframed as "The `gitTag` used for release creation should follow the guidlines of being semver compliant as mentioned in the release creation page. Refer below screenshot"

File: README.md [hunk 3]
```diff
@@ -20,41 +20,16 @@ With Actions Packages, you can reference actions by their version and feel confi
...
+
+<img width="326" alt="Screenshot 2022-11-14 at 5 21 00 PM" src="https://user-images.githubusercontent.com/45332271/201653133-cc78d8b3-c2aa-4009-b190-e569d0814b72.png">
+
+
+
+- Tags that strictly follow semantic versioning(`1.0.0` or `2.3.0`) or tags that have a prefix `v` along with semantic versioned tags(`v1.0.0` or `v2.3.0`) are acceptable (packages cannot be re-published with existing package tags again).
```
Comment (vinaykanamarlapudi) [inline README.md:30]: Can add this as seperate sentence rather in brackets

**Review by vinaykanamarlapudi — CHANGES_REQUESTED** [review]
Added few comments

File: README.md (hunk 1 above)
Comment (aditsrin) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (aditsrin) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (aditsrin) [reply README.md]: Done.

File: README.md [hunk 4]
```diff
@@ -20,41 +20,18 @@ With Actions Packages, you can reference actions by their version and feel confi
...
+
+<img width="326" alt="Screenshot 2022-11-14 at 5 21 00 PM" src="https://user-images.githubusercontent.com/45332271/201653133-cc78d8b3-c2aa-4009-b190-e569d0814b72.png">
+
+
+
+- Tags that strictly follow semantic versioning(`1.0.0` or `2.3.0`) or tags that have a prefix `v` along with semantic versioned tags(`v1.0.0` or `v2.3.0`) are acceptable.
+
+- Packages cannot be re-published with existing package tags again.
+
+- This release.yml is executed when a release is created either through GUI or CLI. Then the user has to track progress of package publishing from the workflow run of this release.yml
```
Comment (vinaykanamarlapudi) [inline README.md:34]: Can be reframed as "Release.yml workflow is triggered when a release is created either through Releases GUI or CLI. The progress of package publishing can be tracked from the workflow run of the release.yml

File: README.md [hunk 5]
```diff
@@ -20,41 +20,18 @@ With Actions Packages, you can reference actions by their version and feel confi
...
+- After adding release.yml, action packages can be created and published to GHCR by following the normal release procedure either using Repo Releases GUI or CLI.
+- The `gitTag` used for release creation should follow the guidlines of being semver compliant as mentioned in the release creation page. Refer below screenshot :
+
+<img width="326" alt="Screenshot 2022-11-14 at 5 21 00 PM" src="https://user-images.githubusercontent.com/45332271/201653133-cc78d8b3-c2aa-4009-b190-e569d0814b72.png">
+
+
+
+- Tags that strictly follow semantic versioning(`1.0.0` or `2.3.0`) or tags that have a prefix `v` along with semantic versioned tags(`v1.0.0` or `v2.3.0`) are acceptable.
+
+- Packages cannot be re-published with existing package tags again.
```
Comment (vinaykanamarlapudi) [inline README.md:32]: Reframed as "As the actions on packages are immutable, packages with existing semver tag once published, they cannot be republished" . 

Check if the above statement is adding more clarity

**Review by vinaykanamarlapudi — APPROVED** [review]

**Pull request merged**